### DISCUSSION TITLE
Got CLI

### DISCUSSION THREAD

**Discussion by Bellisario** [sindresorhus/got#2101] [Ideas]
Would be great if Node.js users were able to use Got as a global installed package and with `npx`.

**Comment by izzqz** [comment]
I'm not confident, this is what Got need to. If you want to make HTTP request from terminal, you can use `curl` and `wget` as well. Otherwise, CLI wrapper of Got will not be better than any terminal HTTP tool.

**Reply by sindresorhus** [reply]
I actually have something for this locally that I use when I cannot know if curl/wget is available. I'll publish something soon. But in general, you should use `curl` whenever possible.

**Reply by izzqz** [reply]
[Windows 8.1 PowerShell has both the `wget` and the `curl` commands.](https://stackoverflow.com/questions/33364752/equivalent-of-wget-command-line-for-windows-8-1/46742133#46742133). And `curl` is relevant from 1996. So, it makes sense only if you are working with legacy environment.

**Reply by Bellisario** [reply]
@izzqz, I know... the problem is that I wanted to use Node.js as a build script for a Go app (I have particular build steps), using just one file without dependencies without any module, because to use them I had to configure Go and Node.js both, with two module files each...

To simplify, I decided to use instead self built functions and some kind of Node.js CLI downloading tool to be cross compatible and prevent additional code import, by using with a simple process spawn.

Anyway, I'll stay tuned about the new @sindresorhus tool!
